src/activities/entities/activity.entity.ts

```diff
@@ -9,17 +9,26 @@ export default class Activities {
   name: string;
 
   @Column()
-  type: string;
+  type: number;
+
+  @Column()
+  destination: number;
 
   @Column({ length: 256 })
   description: string;
 
   @Column()
   cost: string;
 
+  @Column()
+  status: string;
+
   @Column({ type: 'jsonb', nullable: true })
   timeRange: any;
 
   @Column()
   bookingURL: string;
+
+  @Column({ type: 'jsonb', nullable: true })
+  imagesUrl: any;
```

Comment:
Other places its imageUrl. And it's images url. CAN we make this change universal.